front/app/components/CustomFieldsForm/CustomFields.tsx

```diff
@@ -55,24 +56,30 @@ const renderField = ({
           name={question.key}
           hideLocaleSwitcher
           id="e2e-idea-description-input"
+          scrollErrorIntoView={true}
         />
       );
     case 'text':
     case 'number':
       return question.key === 'location_description' ? (
-        <LocationInput name={question.key} />
+        <LocationInput name={question.key} scrollErrorIntoView={true} />
       ) : (
         <Input
           type={question.input_type === 'number' ? 'number' : 'text'}
           name={question.key}
+          scrollErrorIntoView={true}
         />
       );
     case 'multiline_text':
-      return <TextArea name={question.key} />;
+      return <TextArea name={question.key} scrollErrorIntoView={true} />;
     case 'select':
-      return <SingleSelectField question={question} />;
+      return (
+        <SingleSelectField question={question} scrollErrorIntoView={true} />
```

Comment:
For all these fields that are in the `Fields` folder here, and not in the shared `HookForm` folder, we could also define this in the component since this value would always be the `true`. But not super important I guess